providers/teams/lib/apollo.ts

```diff
@@ -0,0 +1,25 @@
+import fetch from 'node-fetch'
+import { ApolloClient, InMemoryCache, NormalizedCacheObject, ApolloClientOptions } from 'apollo-boost'
+import { createHttpLink } from 'apollo-link-http'
+import cookie from 'js-cookie'
+
+const cache = new InMemoryCache({
+    addTypename: false,
+})
+
+const link = createHttpLink({
+    uri: 'http://localhost:2015/graphql',
```

Comment:
This port is in the env file, I think it should be an environment variable in case this will change.